engine/class_modules/priest/sc_priest.hpp

```diff
@@ -606,57 +600,21 @@ struct priest_t final : public player_t
   /**
    * Insanity tracking
    *
-   * Handles the resource gaining from abilities, and insanity draining and manages an event that forcibly punts the
-   * actor out of Voidform the exact moment insanity hits zero (millisecond resolution).
+   * Handles the resource gaining from abilities
    */
   struct insanity_state_t final
   {
-    propagate_const<event_t*> end;             // End event for dropping out of voidform (insanity reaches 0)
-    timespan_t last_drained;  // Timestamp when insanity was last drained
     priest_t& actor;
 
-    const double base_drain_per_sec;
-    const double stack_drain_multiplier;
-    double base_drain_multiplier;
-
     insanity_state_t( priest_t& a );
 
-    // Deferred init for actor dependent stuff not ready in the ctor
     void init();
 
-    // Start the insanity drain tracking
-    void set_last_drained();
-
-    // Start (or re-start) tracking of the insanity drain plus end event
-    void begin_tracking();
-
-    timespan_t time_to_end() const;
-
-    void reset();
-
-    // Compute insanity drain per second with current state of the actor
-    double insanity_drain_per_second() const;
-
     // Gain some insanity
     void gain( double value, gain_t* gain_obj, action_t* source_action = nullptr );
 
     // Lose some insanity
     void lose( double value, gain_t* gain_obj, action_t* source_action = nullptr );
```

Comment:
This seems to have no definition anyway, so the declaration should be removed.
(Has nothing to do with your changes)

Reply:
okay cool, didnt think so but figured id leave it till i got confirmation